components/brave_rewards/resources/page/brave_rewards_page.tsx

```diff
@@ -200,136 +184,131 @@ function reconcileComplete (properties: {type: number, result: number}) {
 
   // EXPIRED TOKEN
   if (properties.result === 24) {
-    getActions().getExternalWallet()
+    actions.getExternalWallet()
   }
 }
 
 function externalWallet (properties: {result: number, wallet: Rewards.ExternalWallet}) {
-  getActions().onExternalWallet(properties.result, properties.wallet)
+  actions.onExternalWallet(properties.result, properties.wallet)
 }
 
 function processRewardsPageUrl (data: Rewards.ProcessRewardsPageUrl) {
-  getActions().onProcessRewardsPageUrl(data)
+  actions.onProcessRewardsPageUrl(data)
 }
 
 function disconnectWallet (properties: {result: number}) {
   if (properties.result === 0) {
-    getActions().getExternalWallet()
-    getActions().getBalance()
+    actions.getExternalWallet()
+    actions.getBalance()
     return
   }
-  getActions().disconnectWalletError()
-}
-
-function onlyAnonWallet (only: boolean) {
-  getActions().onOnlyAnonWallet(only)
+  actions.disconnectWalletError()
 }
 
 function unblindedTokensReady () {
-  getActions().getBalance()
+  actions.getBalance()
 }
 
-function monthlyReport (properties: { result: number, month: number, year: number, report: Rewards.MonthlyReport}) {
-  getActions().onMonthlyReport(properties)
+function monthlyReport (properties: any) {
```

Comment:
Should we define a type for this or it's not worth it?

Reply:
Reverted this change.